beta.go

```diff
@@ -19,7 +17,7 @@ import (
 //
 // See http://mathworld.wolfram.com/BetaFunction.html for more detailed informations.
 func Beta(a, b float64) float64 {
-	return math.Exp(Lbeta(a, b))
+	return gonum.Beta(a, b)
```

Comment:
We probably need a better solution than is here now as this grows, I'm just not sure what the exact strategy is until seeing where the real conflicts are.